todo_app/data/mongo_items.py

```diff
@@ -0,0 +1,64 @@
+from os import stat
+from pydoc import cli
+import pymongo
+from bson.objectid import ObjectId
+import datetime
+
+from todo_app.data.item import Item
+
+class MongoItems():
+    def __init__(self, config):
+        self._config = config
+
+    def with_auth_params(self, params):
+        return { **self.get_auth_params(), **params }
```

Comment:
Is this just copied over from Trello? I can't see that this is used, or that the `get_auth_params` method is defined